### PR TITLE
Add city area api

### PR DESCRIPTION
**Pull request by BeshoyMelika**

### CODE REVIEW

File: lib/apis/user_api_manager.dart [hunk 1]
```diff
@@ -127,4 +128,26 @@ class UserApiManager extends BaseApiManager {
       success(GetAllAddress.fromJson(result.data).addresses);
     }
   }
+
+  static Future<void> getAllCities(Function success, Function fail) async {
```
Comment (MinaSamirCap) [inline lib/apis/user_api_manager.dart]: I totally recommend separating addresses APIs and City Area APIs 
each of them in a stand-alone file

**Review by MinaSamirCap — CHANGES_REQUESTED** [review]
kindly check the comments and push your changes asap

File: lib/apis/user_api_manager.dart (hunk 1 above)
Comment (BeshoyMelika) [reply lib/apis/user_api_manager.dart]: I separated addresses APIs to AddressApiManager
and  City Area APIs to AddressAssetApiManager

File: lib/apis/adderss_asset_api_manager.dart [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import 'package:graphql_flutter/graphql_flutter.dart';
+
+import '../generatedql/address_asset/graphql_api.dart';
+import './base_api_manager.dart';
+
+class AddressAssetApiManager extends BaseApiManager {
```
Comment (MinaSamirCap) [inline lib/apis/adderss_asset_api_manager.dart]: add these to the address api manager not need for more classes

**Review by MinaSamirCap — CHANGES_REQUESTED** [review]

**Review by MinaSamirCap — APPROVED** [review]

**Pull request merged**